lib/mock/session.ex

```diff
@@ -0,0 +1,76 @@
+defmodule Janus.Mock.Session do
+  @moduledoc """
+  Provides a way to prop reponses from `Janus.Session` module.
```

Comment:
Is that what you meant?
```suggestion
  Provides a way to mock reponses from `Janus.Session` module.
```